src/app/components/logIn/Login.tsx

```
@@ -23,51 +24,98 @@ const Login = () => {
+		validateInput(id);
+		if (id === '') {
+			setErrorId('');
+		}
+	};
+
+	//useEffect: ID 변경 후 1초 지났을 때 유효성 검사 시작한다
+	useEffect(() => {
+		const timer = setTimeout(() => {
+			validateInput(id);
+		}, 1000);
+		console.log('errror: ', errorId);
```

ESLint

[error] Unexpected console statement.

(no-console)